### DISCUSSION TITLE
Can't find Code Sniffer ruleset, it's a bit too strict

### DISCUSSION THREAD

**Discussion by Hnnx** [ixkaito/bathe#39] [Q&A] [answered]
Hi,
First of all I'd like to thank everyone for creating this starter theme that works great out of the box.

I would like to omit some code sniffer rules but can't figure out how to do it. I've seen some tutorials where you can create ruleset.xml and than add/remove specific parts but I'm not sure if it's running on server or if one of the VSC extensions is highlighting errors.

Any help is greatly appreciated

**Comment by ixkaito** [comment]
@Hnnx Thank you for your report! Seems I forgot to update some documentation. I'll inform you again.

**Comment by ixkaito** [comment, marked as answer]
@Hnnx I updated the [documentation](https://ixkaito.github.io/bathe/). (Quick Start No.5)

If you want to use PHP CodeSniffer, you have to `composer install` first. Then the rule set will be installed to the `vendor` folder.

To customize the rules, create a `phpcs.xml` file and add some codes like the following:

```xml
<?xml version="1.0"?>
<ruleset name="My WordPress">
    <rule ref="WordPress">
        <exclude name="Universal.Operators.DisallowShortTernary"/>
    </rule>
</ruleset>

```
